01-getting-started/002-env-create.md

```diff
@@ -62,41 +62,97 @@ The namespaces directory contains a directory for each of the clusters that you
+To set up an environment we create 5 files in the directory for your namespace:
 
 * [`00-namespace.yaml`](#00-namespaceyaml)
 * [`01-rbac.yaml`](#01-rbacyaml)
 * [`02-limitrange.yaml`](#02-limitrangeyaml)
 * [`03-resourcequota.yaml`](#03-resourcequotayaml)
 * [`04-networkpolicy.yaml`](#04-networkpolicyyaml)
 
-These files define key elements of the namespace and restrictions we want to place on it so that we have security and resource allocation properties. We describe each of these files in more detail below.
+These files define key elements of the namespace and restrictions we want to place on it so that we have security and resource allocation properties. We will use terraform to create these files from templates. We also describe each of these files [in more detail below](#00-namespaceyaml) in case you want to make future changes.
+
+### Create your namespace and namespace resources
+
+We automate the creation of the namespace resource files using terraform. You will need to install terraform locally:
+
+```Shell
+$ brew install terraform
+```
+
+In each of the template files you need to replace some example values with information about your namespace, team or app. We do this by running terraform commands and filling in the values.
+
+These are the inputs for the terraform module, that you will need to fill:
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| application |  | string | - | yes |
+| business-unit | Area of the MOJ responsible for the service | string | - | yes |
+| cluster | What cluster are you deploying your namespace. i.e cloud-platform-test-1 | string | `cloud-platform-live-0` | no |
+| contact_email | Contact email address for owner of the application | string | - | yes |
+| environment |  | string | - | yes |
+| github_team | This is your team name as defined by the GITHUB api. This has to match the team name on the Github API | string | - | yes |
+| is-production |  | string | `false` | no |
+| namespace | Namespace you would like to create on cluster <application>-<environment>. i.e myapp-dev | string | - | yes |
+| owner | Who is the owner/Who is responsible for this application | string | - | yes |
+| source_code_url | Url of the source code for your application | string | - | yes |
+
+Create your namespace and these resources files, filling in the values as you are prompted:
+
+```Shell
+$ cd cloud-platform-environments/namespace-resources/
+$ terraform init
+$ terraform plan
+$ terraform apply
+```
 
-To get started copy the example versions of these files from the [`namespace-resources`](https://github.com/ministryofjustice/cloud-platform-environments/tree/master/namespace-resources) directory in the `cloud-platform-evironments` repo into your `<servicename-env>` directory.
+Our terraform module creates a namespace on live-0 by default but if you would like to deploy to another cluster you can use:
```

Comment:
this might cause confusion: resources are not created directly in any cluster, just the definition written to local files which then need to be PRd

Reply:
I'll make it "the files for a new namespace..."